tests/chainer_tests/initializer_tests/test_normal.py

```diff
@@ -62,6 +63,8 @@ def setUp(self):
             kwargs['scale'] = self.scale
         if self.fan_option is not None:
             kwargs['fan_option'] = self.fan_option
+        if self.rng is not None:
+            kwargs['rng'] = self.rng
```

Comment:
How about
```
        if self.rng_class is not None:
            kwargs['rng'] = self.rng_class()
```
with `'rng_class': [None, numpy.random.RandomState],` in order to avoid sharing rng among tests?  It also fixes the issue that the names of the testing class are not stable (because of #5782).

Reply:
I agree to the suggestion.